owncloudApp/src/main/java/com/owncloud/android/operations/SyncProfileOperation.kt

```diff
@@ -60,30 +60,33 @@ class SyncProfileOperation(
                         setUserData(account, AccountUtils.Constants.KEY_ID, userInfo.id)
                     }
 
-                    val refreshUserQuotaFromServerAsyncUseCase: RefreshUserQuotaFromServerAsyncUseCase by inject()
-                    val userQuotaResult =
-                        refreshUserQuotaFromServerAsyncUseCase(
-                            RefreshUserQuotaFromServerAsyncUseCase.Params(
-                                account.name
-                            )
-                        )
-                    userQuotaResult.getDataOrNull()?.let {
-                        Timber.d("User quota synchronized for account ${account.name}")
+                    val getStoredCapabilitiesUseCase: GetStoredCapabilitiesUseCase by inject()
+                    val storedCapabilities = getStoredCapabilitiesUseCase(GetStoredCapabilitiesUseCase.Params(account.name))
 
-                        val getStoredCapabilitiesUseCase: GetStoredCapabilitiesUseCase by inject()
+                    storedCapabilities?.let {
+                        if (it.isSpacesAllowed()) {
+                            val refreshUserQuotaFromServerAsyncUseCase: RefreshUserQuotaFromServerAsyncUseCase by inject()
+                            val userQuotaResult =
+                                refreshUserQuotaFromServerAsyncUseCase(
+                                    RefreshUserQuotaFromServerAsyncUseCase.Params(
+                                        account.name
+                                    )
+                                )
+                            userQuotaResult.getDataOrNull()?.let {
+                                Timber.d("User quota synchronized for account ${account.name}")
```

Comment:
Just to make clear the difference in the logs, I would add an oC10 mention here:
```suggestion
                                Timber.d("User quota synchronized for oC10 account ${account.name}")
```